docs/_sources/meta.rst.txt

```diff
@@ -356,7 +356,7 @@ Normally Python packages should use this line:
 .. code-block:: yaml
 
     build:
-      script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+      script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv"
```

Comment:
Can we remove `--no-deps` and `--ignore-installed` as well? conda-build sets env variables that does the same thing right?

Reply:
Even though I prefer the explicitness of the having the options in there, since they are already in `conda-build` (and my guess is that they won't remove/change them) I'm +0.5.